examples/docker/modin-ray/plasticc.py

```diff
@@ -230,7 +230,7 @@ def ml(train_final, test_final):
         evals=watchlist,
         feval=func_loss,
         early_stopping_rounds=10,
-        verbose_eval=None,
+        verbose_eval=1000,
```

Comment:
```suggestion
        verbose_eval=None,
```
Decided to leave as is to not issue a lot of warnings which might be kind of confusing to users.